.github/ISSUE_TEMPLATE/issue_report.yml

```
@@ -18,17 +18,13 @@ description: "File an issue report to make us aware of a bug or issue."
 body:
   - type: markdown
     attributes:
-      value: Thanks for taking the time to fill out this issue report! Don't forget to give it a meaningful title so users can find the report easily.
-  - type: checkboxes
-    id: not_an_exploit
-    attributes:
-      label: ⚠️ Could this issue be an exploit? ⚠️
-      description: "[If your issue is an ingame exploit that allows players to gain an unfair advantage, or could affect server stability or security, please file it on the forum here.](https://www.paradisestation.org/forum/179-exploit-reports/)"
-      options:
-        - label: I confirm this issue does not give players an unfair advantage. (Required)
-          required: true
-        - label: I confirm this issue does not affect server stability or security. (Required)
-          required: true
+    value: >
+      Thank you for taking the time to complete this issue report.
+      Please ensure the title is clear and descriptive to help others
+      locate and understand the report more easily.
+
+      If you believe this is an exploit, please [report it on the forums instead](https://www.paradisestation.org/forum/179-exploit-reports/).
```

```suggestion
      If you believe this is an exploit(E.G, something that could give you an unfair advantage over others), please [report it on the forums instead](https://www.paradisestation.org/forum/179-exploit-reports/).
```
Unsure if this has been discussed else where, but i feel like we should be clear on what *is* an exploit with these

---

Even with this suggestion, I'm not sure what counts as an exploit. Do we have a page that details some exploit examples?